src/riot-rs-threads/src/lib.rs

```diff
@@ -321,6 +321,12 @@ pub fn wakeup(thread_id: ThreadId) -> bool {
     })
 }
 
+/// Returns the size of the internal structure that holds the
+/// a thread's data.
+pub fn thread_struct_size() -> usize {
```

Comment:
Name this `tcb_size`? `Thread` is basically the TCB, right?

Reply:
(maybe keep this out of the docs so people won't rely on it. :smile:)